source/frontend/src/assets/scss/_variables.scss

```diff
@@ -2,4 +2,4 @@
 
 $header-height: 7.2rem;
 $mapfilter-height: 6rem;
-$footer-height: 3.4rem;
+$footer-height: 4.4rem;
```

Comment:
I don't think this is what Ana wanted, please check with her.

Reply:
I checked with her and she says that is ok to increase the footer height as long as the bomb icon is as the size as the help icon. I increased a little bit more than that since the icon looked too small. I showed to her and she was good with the changes.

<img width="826" alt="image" src="https://github.com/user-attachments/assets/2c062f90-0332-432e-8e2e-5d4937e6b3b7">